pkg/config/config.go

```diff
@@ -1053,7 +1053,15 @@ func (c *Config) Write() error {
 	if err := os.MkdirAll(filepath.Dir(path), 0755); err != nil {
 		return err
 	}
-	configFile, err := os.OpenFile(path, os.O_CREATE|os.O_RDWR|os.O_TRUNC, 0600)
+	// If we're creating /etc/containers/containers.conf,
+	// make sure it can be read by both root and rootless podman
+	var configPerms os.FileMode
+	if path == OverrideContainersConfig {
+		configPerms = 0644
+	} else {
+		configPerms = 0600
```

Comment:
If its a non-root user, this will create $HOME/.config/containers/containers.conf, or whatever $CONTAINERS_CONF is set to and I think we wouldn't want other users to see this conf? But since its in a users homedir, I'd be okay with always using 0644, what do you think?

Reply:
Yeah, I think since it's *just* a configuration file with no sensitive information a 0644 should be fine.